packages/lib-user/src/components/UserHome/components/RecentSubjects/RecentSubjectsContainer.js

```diff
@@ -0,0 +1,37 @@
+import useSWR from 'swr'
+import { shape, string } from 'prop-types'
+
+import fetchRecentSubjects from './fetchRecentSubjects.js'
+import RecentSubjects from './RecentSubjects.js'
+
+const SWROptions = {
+  revalidateIfStale: true,
+  revalidateOnMount: true,
+  revalidateOnFocus: true,
+  revalidateOnReconnect: true,
+  refreshInterval: 0
+}
+
+function RecentSubjectsContainer({ authUser }) {
+  const cacheKey = {
+    name: 'user-recent-classifications',
+    userId: authUser.id
+  }
+  const {
+    data: subjects,
```

Comment:
⚠️ The data being fetches by fetchRecentSubjects() is actually **classification resources,** not **subject resources.**

For example, "subjects" here might actually look something like...

```
[
  {
    id: "258337208",  // Classification ID
    links: {
      project: "19072",
      workflow: "22152",
      subject: "80286011"  // Actual Subject ID
    },
    ...
  },
...
]
```

Reply:
That’s not a Classification, it’s a Recent, which is a Panoptes resource representing your recent subjects.

Reply:
There’s some code in the project app that uses these for the Classify page.

Reply:
Correct - are you suggesting code changes in this file?

Reply:
I don’t think so. Just clarifying that the ID isn’t a classification ID or a subject ID.

Reply:
Oh, I see, you call them `classification` in the code. The code works, but the name might be confusing if someone expects them to be Classification objects.

In the project app, they're named `recent`. It's a tiny thing, but might make the code easier to follow.

https://github.com/zooniverse/front-end-monorepo/blob/4deea7e0476c7352e768da6eaadfed1ee0ccf169/packages/app-project/stores/User/Recents/Recents.js#L48-L54

Reply:
Gotcha, thanks!